### ISSUE TITLE
Bower Install Issue

### ISSUE THREAD

**Issue by ThomasDolso** [cloudinary/cloudinary_angular#38]
Hi,
when I try to install the bower components for the photoalbum I get the following error:

bower ng-file-upload#7.0.x      cached git://github.com/danialfarid/angular-file-upload-bower.git#7.0.17
bower ng-file-upload#7.0.x    validate 7.0.17 against git://github.com/danialfarid/angular-file-upload-bower.git#7.0.x
bower angular-route#1.4.x       cached git://github.com/angular/bower-angular-route.git#1.4.6
bower angular-route#1.4.x     validate 1.4.6 against git://github.com/angular/bower-angular-route.git#1.4.x
bower angular#1.4.x             cached git://github.com/angular/bower-angular.git#1.4.6
bower angular#1.4.x           validate 1.4.6 against git://github.com/angular/bower-angular.git#1.4.x
bower angular-animate#1.4.x     cached git://github.com/angular/bower-angular-animate.git#1.4.6
bower angular-animate#1.4.x   validate 1.4.6 against git://github.com/angular/bower-angular-animate.git#1.4.x
bower angular-resource#1.4.x    cached git://github.com/angular/bower-angular-resource.git#1.4.6
bower angular-resource#1.4.x  validate 1.4.6 against git://github.com/angular/bower-angular-resource.git#1.4.x
bower cloudinary_ng#master     ENORESTARGET File system sources can't resolve targets

Could you please tell me what's wrong?
Thanks
Tom

**Comment by ArdentKid** [comment]
bump.

getting the same message, won't get through bower install or npm start.

**Comment by ArdentKid** [comment]
here's my npm-debug.log:

```
0 info it worked if it ends with ok
1 verbose cli [ 'node', '/usr/local/bin/npm', 'start' ]
2 info using npm@2.11.0
3 info using node@v0.12.2
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info prestart angular-cloudinary-photo@0.0.0
6 verbose unsafe-perm in lifecycle true
7 info angular-cloudinary-photo@0.0.0 Failed to exec prestart script
8 verbose stack Error: angular-cloudinary-photo@0.0.0 prestart: `npm install`
8 verbose stack Exit status 1
8 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:213:16)
8 verbose stack     at EventEmitter.emit (events.js:110:17)
8 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:24:14)
8 verbose stack     at ChildProcess.emit (events.js:110:17)
8 verbose stack     at maybeClose (child_process.js:1015:16)
8 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
9 verbose pkgid angular-cloudinary-photo@0.0.0
10 verbose cwd /Users/Omid/Downloads/cloudinary_angular-master/samples/photo_album
11 error Darwin 14.3.0
12 error argv "node" "/usr/local/bin/npm" "start"
13 error node v0.12.2
14 error npm  v2.11.0
15 error code ELIFECYCLE
16 error angular-cloudinary-photo@0.0.0 prestart: `npm install`
16 error Exit status 1
17 error Failed at the angular-cloudinary-photo@0.0.0 prestart script 'npm install'.
17 error This is most likely a problem with the angular-cloudinary-photo package,
17 error not with npm itself.
17 error Tell the author that this fails on your system:
17 error     npm install
17 error You can get their info via:
17 error     npm owner ls angular-cloudinary-photo
17 error There is likely additional logging output above.
18 verbose exit [ 1, true ]
```

**Comment by ArdentKid** [comment]
@ThomasDolso found the solution. In samples/photo_album/bower.json, change your cloudinary_ng dependency to just "#master":

```
{
  "name": "cloudinary-photo-album",
  "description": "A starter project for Cloudinary + AngularJS",
  "version": "0.0.0",
  "homepage": "https://github.com/angular/angular-seed",
  "license": "MIT",
  "private": true,
  "dependencies": {
    "ng-file-upload": "7.0.x",
    "angular": "1.4.x",
    "angular-route": "1.4.x",
    "angular-resource": "1.4.x",
    "angular-animate": "1.4.x",
    "cloudinary_ng": "#master"
  },
  "ignore": {}
}
```

**Comment by tocker** [comment]
Fixed by #40. Thanks!
